Kickstarter-iOS/DataSources/MessageThreadsDataSource.swift

```diff
@@ -14,7 +14,7 @@ internal final class MessageThreadsDataSource: ValueCellDataSource {
   }
 
   internal func emptyState(isVisible: Bool) {
-    self.set(cellIdentifiers: isVisible ? ["MessageThreadsEmptyState"] : [],
+    self.set(cellIdentifiers: isVisible ? ["MessageThreadEmptyStateCell"] : [],
```

Comment:
Possible to write a test? Also would be nice to throw this in an enum or constant.